src/hooks/useSykefravarsstatistikk.ts

```diff
@@ -0,0 +1,68 @@
+import { RestAltinnOrganisasjoner } from '../api/altinnorganisasjon-api';
+import { useAltinnOrganisasjoner } from './useAltinnOrganisasjoner';
+import { useAltinnOrganisasjonerMedStatistikk } from './useAltinnOrganisasjonerMedStatistikk';
+import { RestVirksomhetsdata } from '../api/virksomhetsdata-api';
+import { useVirksomhetsdata } from './useVirksomhetsdata';
+import { EnhetsregisteretState, useEnheter } from './useEnheter';
+import { RestSummertSykefraværshistorikk } from '../api/summert-sykefraværshistorikk-api';
+import { useSummertSykefravær } from './useSummertSykefravær';
+import { useSykefraværshistorikk } from './useSykefraværshistorikk';
+import { RestSykefraværshistorikk } from '../api/kvartalsvis-sykefraværshistorikk-api';
+import { useFeatureToggles } from './useFeatureToggles';
+import { RestFeatureToggles } from '../api/feature-toggles-api';
+import {
+    Ekstradata,
+    getEkstraDataFraEnhetsregisteret,
+    getEkstraDataFraSummertSykefraværshistorikk,
+    getEkstraDataFraSykefraværshistorikk,
+    getEkstraDataFraVirksomhetsdata,
+} from '../amplitude/ekstradata';
+
+export interface SykefraværAppData {
+    altinnOrganisasjoner: RestAltinnOrganisasjoner;
+    altinnOrganisasjonerMedStatistikk: RestAltinnOrganisasjoner;
+    virksomhetsdata: RestVirksomhetsdata;
+    enhetsInformasjon: EnhetsregisteretState;
```

Comment:
kunne vi kalle det for `enhetsRegisteretInformasjon`? Blir litt lettere å forvente hva det kan inneholde.